lib/shared/movie.dart

```diff
@@ -8,51 +11,86 @@ class Movie {
     required this.genres,
     required this.overview,
     this.saved = false,
-    this.cast = const <String>[],
+    required this.cast,
   });
 
+  final int id;
   final String title;
   final String posterName;
   final String backdropName;
   final String releaseDate;
-  final String genres;
+  final List<int> genres;
   final String overview;
   final double score;
-  final List<String> cast;
+  final List<dynamic> cast;
```

Comment:
Avoid the usage of `dynamic` keyword when you do know the content of the payload. You could create a `Cast` data class for this

Reply:
It only have an atribute String to define it path, it's okay anyway? Meybe i create for extensibility although i won't use it in this app. Thoughts?

Reply:
Fixed